### PR TITLE
Refactor tests

### PR DESCRIPTION
**Pull request by auhc99**
Refactored tests for logic and model, removing most magic strings

### CODE REVIEW

File: src/test/java/seedu/anilist/logic/commands/CommandResultTest.java [hunk 1]
```diff
@@ -13,23 +11,26 @@ public void equals() {
...
         // different feedbackToUser value -> returns false
-        assertFalse(commandResult.equals(new CommandResult("different")));
+        assertNotEquals(commandResult, new CommandResult("different"));
 
         // different exit value -> returns false
-        assertFalse(commandResult.equals(new CommandResult("feedback", false, true)));
+        assertNotEquals(commandResult, new CommandResult("feedback", false, true));
+
+        // different stats value -> returns false
+        assertNotEquals(commandResult, new CommandResult("feedback", true, false));
```
Comment (clyveycui) [inline src/test/java/seedu/anilist/logic/commands/CommandResultTest.java:33]: A bit of magic strings used here, perhaps can put "feedback" as a constant?

File: src/test/java/seedu/anilist/logic/commands/CommandTestUtil.java [hunk 2]
```diff
@@ -31,50 +30,128 @@
...
-    public static final String VALID_STATUS_TOWATCH = "towatch";
-    public static final String VALID_STATUS_WATCHING = "watching";
 
-    public static final String INVALID_ACTION_EMPTY_STRING = "";
-    public static final String INVALID_EPISODE_LARGER_THAN_MAX_INT =
-        Long.toString((long) Integer.MAX_VALUE + 1);
-    public static final String INVALID_ACTION_NO_SUCH_ACTION = "dancing";
-    public static final String INVALID_GENRE_NOT_IN_LIST = "tentacles";
+    // Invalid Names
+    public static final String INVALID_NAME_LONGER_THAN_MAX_LENGTH = "F".repeat(142);
```
Comment (clyveycui) [inline src/test/java/seedu/anilist/logic/commands/CommandTestUtil.java:50]: the max length here you can probably import a constant from the Name class

File: src/test/java/seedu/anilist/logic/commands/CommandsListTest.java [hunk 3]
```diff
@@ -40,9 +40,9 @@ public void getListOfCommands() {
      */
     private String listOfCommandsToString(String[] listOfCommands) {
         String result = "";
-        for (int i = 0; i < listOfCommands.length; i++) {
+        for (String listOfCommand : listOfCommands) {
```
Comment (clyveycui) [inline src/test/java/seedu/anilist/logic/commands/CommandsListTest.java:43]: uhh shouldn't it be
command instead of listOfCommand

File: src/test/java/seedu/anilist/logic/commands/ConfirmClearCommandTest.java [hunk 4]
```diff
@@ -22,7 +22,7 @@
 public class ConfirmClearCommandTest {
 
     /** {@code Predicate} that always evaluate to false */
-    private Predicate<Anime> predicateShowNoAnime = unused -> false;
+    private final Predicate<Anime> predicateShowNoAnime = unused -> false;
```
Comment (clyveycui) [inline src/test/java/seedu/anilist/logic/commands/ConfirmClearCommandTest.java:25]: final should be all caps

File: src/test/java/seedu/anilist/model/genre/GenreTest.java [hunk 5]
```diff
@@ -33,25 +39,24 @@ public void isValidGenreName() {
...
+        assertFalse(Genre.isValidGenreName(INVALID_GENRE_NON_ALPHANUMERIC));
 
         // valid genres
-        // genre with lower case
-        assertTrue(Genre.isValidGenreName(VALID_GENRE_LOWER_CASE));
-        // genre with mixed case
-        assertTrue(Genre.isValidGenreName(VALID_GENRE_MIXED_CASE));
+        assertTrue(Genre.isValidGenreName(VALID_GENRE_ACTION));
+        assertTrue(Genre.isValidGenreName(VALID_GENRE_SCIENCE_FICTION));
+        assertTrue(Genre.isValidGenreName(VALID_GENRE_SUPERNATURAL));
```
Comment (clyveycui) [inline src/test/java/seedu/anilist/model/genre/GenreTest.java:52]: doesn't really show the equivalence partitioning here anymore, but I guess it is inside the commandTestUtil instead?

Comment (clyveycui) [reply src/test/java/seedu/anilist/model/genre/GenreTest.java:52]: might want to add comments here anyways to show the equivalence partitioning

File: src/test/java/seedu/anilist/model/anime/StatusTest.java [hunk 6]
```diff
@@ -25,17 +38,17 @@ public void isValidStatus() {
...
-        assertTrue(Status.isValidStatus("watching")); // recognised lowercase
-        assertTrue(Status.isValidStatus("FINISHED")); // recognised uppercase
-        assertTrue(Status.isValidStatus("toWATcH")); // recognised mixed case
-        assertTrue(Status.isValidStatus("t")); // recognised shortform lowercase
-        assertTrue(Status.isValidStatus("W")); // recognised shortform uppercase
+        assertTrue(Status.isValidStatus(VALID_STATUS_WATCHING));
+        assertTrue(Status.isValidStatus(VALID_STATUS_WATCHING_SHORT_FORM));
+        assertTrue(Status.isValidStatus(VALID_STATUS_TOWATCH));
+        assertTrue(Status.isValidStatus(VALID_STATUS_TOWATCH_SHORT_FORM));
+        assertTrue(Status.isValidStatus(VALID_STATUS_FINISHED));
```
Comment (clyveycui) [inline src/test/java/seedu/anilist/model/anime/StatusTest.java:52]: probably don't need so many testcases

File: src/test/java/seedu/anilist/model/anime/StatusEqualsPredicateTest.java [hunk 7]
```diff
@@ -13,31 +16,31 @@
     @Test
     public void equals() {
         Status firstStatus = new Status(VALID_STATUS_TOWATCH);
-        Status secondStatus = new Status(VALID_STATUS_WATCHING);
+        Status secondStatus = new Status(VALID_STATUS_FINISHED);
```
Comment (clyveycui) [inline src/test/java/seedu/anilist/model/anime/StatusEqualsPredicateTest.java:19]: why this change lol?

File: src/test/java/seedu/anilist/model/anime/NameTest.java [hunk 8]
```diff
@@ -26,22 +38,18 @@ public void isValidName() {
...
-        assertTrue(Name.isValidName("Higehiro: After Being Rejected, I Shaved and Took in a High School Runaway"));
-        assertTrue(Name.isValidName(VALID_NAME_MAX_LENGTH)); // max length
-        assertTrue(Name.isValidName("Gintama'")); // name containing apostrophe
+        assertTrue(Name.isValidName(VALID_NAME_BNHA));
+        assertTrue(Name.isValidName(VALID_NAME_AKIRA));
+        assertTrue(Name.isValidName(VALID_NAME_EIGHTY_SIX));
+        assertTrue(Name.isValidName(VALID_NAME_FATE_ZERO));
+        assertTrue(Name.isValidName(VALID_NAME_JOJO_GOLDEN_WIND));
+        assertTrue(Name.isValidName(VALID_NAME_NON_ALPHANUMERIC_ASCII));
+        assertTrue(Name.isValidName(VALID_NAME_MAX_LENGTH));
```
Comment (clyveycui) [inline src/test/java/seedu/anilist/model/anime/NameTest.java:53]: might want to add comments to make equivalence partitioning more clear
either add in each test file, or add it inside command test utils

File: src/test/java/seedu/anilist/model/anime/EpisodeTest.java [hunk 9]
```diff
@@ -14,29 +26,36 @@ public void constructor_null_throwsNullPointerException() {
...
+        assertFalse(Episode.isValidEpisode(INVALID_EPISODE_DECIMAL));
+        assertFalse(Episode.isValidEpisode(INVALID_EPISODE_NEG));
+        assertFalse(Episode.isValidEpisode(INVALID_EPISODE_LARGER_THAN_MAX_EPISODE));
+        assertFalse(Episode.isValidEpisode(INVALID_EPISODE_LARGER_THAN_MAX_INT));
+
+        // valid episodes
+        assertTrue(Episode.isValidEpisode(VALID_EPISODE_ZERO));
+        assertTrue(Episode.isValidEpisode(VALID_EPISODE_ONE));
+        assertTrue(Episode.isValidEpisode(VALID_EPISODE_TWO));
+        assertTrue(Episode.isValidEpisode(VALID_EPISODE_MAX));
```
Comment (clyveycui) [inline src/test/java/seedu/anilist/model/anime/EpisodeTest.java:58]: probably can reduce the test cases here

File: src/test/java/seedu/anilist/model/ModelManagerTest.java [hunk 10]
```diff
@@ -118,13 +118,13 @@ public void equals() throws ParseException {
...
-        assertTrue(modelManager.equals(modelManagerCopy));
+        assertEquals(modelManager, modelManagerCopy);
 
         // same object -> returns true
-        assertTrue(modelManager.equals(modelManager));
+        assertEquals(modelManager, modelManager);
 
         // null -> returns false
-        assertFalse(modelManager.equals(null));
+        assertFalse(modelManager == null);
```
Comment (clyveycui) [inline src/test/java/seedu/anilist/model/ModelManagerTest.java:127]: maybe use assert not equals here

File: src/test/java/seedu/anilist/logic/parser/UpdateStatusCommandParserTest.java [hunk 11]
```diff
@@ -24,7 +24,7 @@
     private static final String MESSAGE_INVALID_FORMAT =
             String.format(MESSAGE_INVALID_COMMAND_FORMAT, UpdateStatusCommand.MESSAGE_USAGE);
 
-    private UpdateStatusCommandParser parser = new UpdateStatusCommandParser();
+    private final UpdateStatusCommandParser parser = new UpdateStatusCommandParser();
```
Comment (clyveycui) [inline src/test/java/seedu/anilist/logic/parser/UpdateStatusCommandParserTest.java:27]: final variable should be all caps

File: src/test/java/seedu/anilist/logic/parser/UpdateEpisodeCommandParserTest.java [hunk 12]
```diff
@@ -24,7 +24,7 @@
     private static final String MESSAGE_INVALID_FORMAT =
         String.format(MESSAGE_INVALID_COMMAND_FORMAT, UpdateEpisodeCommand.MESSAGE_USAGE);
 
-    private UpdateEpisodeCommandParser parser = new UpdateEpisodeCommandParser();
+    private final UpdateEpisodeCommandParser parser = new UpdateEpisodeCommandParser();
```
Comment (clyveycui) [inline src/test/java/seedu/anilist/logic/parser/UpdateEpisodeCommandParserTest.java:27]: final variable should be all caps

File: src/test/java/seedu/anilist/logic/parser/RenameCommandParserTest.java [hunk 13]
```diff
@@ -22,7 +22,7 @@
     private static final String MESSAGE_INVALID_FORMAT =
         String.format(MESSAGE_INVALID_COMMAND_FORMAT, RenameCommand.MESSAGE_USAGE);
 
-    private RenameCommandParser parser = new RenameCommandParser();
+    private final RenameCommandParser parser = new RenameCommandParser();
```
Comment (clyveycui) [inline src/test/java/seedu/anilist/logic/parser/RenameCommandParserTest.java:25]: final variable should be all caps

File: src/test/java/seedu/anilist/logic/parser/ParserUtilTest.java [hunk 14]
```diff
@@ -165,37 +186,42 @@ public void parseGenres_emptyCollection_returnsEmptySet() throws Exception {
...
-        Set<Genre> expectedGenreSet = new HashSet<Genre>(
-                Arrays.asList(new Genre(VALID_GENRE_1), new Genre(VALID_GENRE_2))
+        Set<Genre> actualGenreSet = ParserUtil.parseGenres(Arrays.asList(VALID_GENRE_ACTION,
+                VALID_GENRE_SCIENCE_FICTION));
+        Set<Genre> expectedGenreSet = new HashSet<>(
+                Arrays.asList(new Genre(VALID_GENRE_ACTION),
+                        new Genre(VALID_GENRE_SCIENCE_FICTION.toLowerCase()))
         );
-
+        System.out.println("A: ");
```
Comment (clyveycui) [inline src/test/java/seedu/anilist/logic/parser/ParserUtilTest.java:195]: what's this for?

File: src/test/java/seedu/anilist/logic/parser/ListCommandParserTest.java [hunk 15]
```diff
@@ -20,7 +20,7 @@
 public class ListCommandParserTest {
     private static final String MESSAGE_INVALID_FORMAT =
             String.format(MESSAGE_INVALID_COMMAND_FORMAT, ListCommand.MESSAGE_USAGE);
-    private ListCommandParser parser = new ListCommandParser();
+    private final ListCommandParser parser = new ListCommandParser();
```
Comment (clyveycui) [inline src/test/java/seedu/anilist/logic/parser/ListCommandParserTest.java:23]: final variable should be all caps

File: src/test/java/seedu/anilist/logic/parser/GenreCommandParserTest.java [hunk 16]
```diff
@@ -22,15 +22,14 @@
...
-import seedu.anilist.model.anime.Anime;
 import seedu.anilist.model.genre.Genre;
 import seedu.anilist.testutil.GenresDescriptorBuilder;
 
 public class GenreCommandParserTest {
     private static final String MESSAGE_INVALID_FORMAT =
             String.format(MESSAGE_INVALID_COMMAND_FORMAT, GenreCommand.MESSAGE_USAGE);
 
-    private GenreCommandParser parser = new GenreCommandParser();
+    private final GenreCommandParser parser = new GenreCommandParser();
```
Comment (clyveycui) [inline src/test/java/seedu/anilist/logic/parser/GenreCommandParserTest.java:32]: final variable should be all caps

File: src/test/java/seedu/anilist/logic/parser/FindCommandParserTest.java [hunk 17]
```diff
@@ -16,7 +16,7 @@
 
 public class FindCommandParserTest {
 
-    private FindCommandParser parser = new FindCommandParser();
+    private final FindCommandParser parser = new FindCommandParser();
```
Comment (clyveycui) [inline src/test/java/seedu/anilist/logic/parser/FindCommandParserTest.java:19]: final variable should be all caps

File: src/test/java/seedu/anilist/logic/parser/FindCommandParserIntegrationTest.java [hunk 18]
```diff
@@ -38,16 +38,16 @@
     private static final String MESSAGE_INVALID_FORMAT =
         String.format(MESSAGE_INVALID_COMMAND_FORMAT, FindCommand.MESSAGE_USAGE);
 
-    private Model model = new ModelManager(getTypicalAnimeList(), new UserPrefs());
-    private Model expectedModel = new ModelManager(getTypicalAnimeList(), new UserPrefs());
-    private FindCommandParser parser = new FindCommandParser();
+    private final Model model = new ModelManager(getTypicalAnimeList(), new UserPrefs());
+    private final Model expectedModel = new ModelManager(getTypicalAnimeList(), new UserPrefs());
+    private final FindCommandParser parser = new FindCommandParser();
```
Comment (clyveycui) [inline src/test/java/seedu/anilist/logic/parser/FindCommandParserIntegrationTest.java:43]: final variable should be all caps

File: src/test/java/seedu/anilist/logic/commands/UpdateStatusCommandTest.java [hunk 19]
```diff
@@ -31,7 +31,7 @@
  */
 public class UpdateStatusCommandTest {
 
-    private Model model = new ModelManager(getTypicalAnimeList(), new UserPrefs());
+    private final Model model = new ModelManager(getTypicalAnimeList(), new UserPrefs());
```
Comment (clyveycui) [inline src/test/java/seedu/anilist/logic/commands/UpdateStatusCommandTest.java:34]: final variable should be all caps

File: src/test/java/seedu/anilist/logic/parser/AddCommandParserTest.java [hunk 20]
```diff
@@ -36,7 +36,7 @@
 import seedu.anilist.testutil.AnimeBuilder;
 
 public class AddCommandParserTest {
-    private AddCommandParser parser = new AddCommandParser();
+    private final AddCommandParser parser = new AddCommandParser();
```
Comment (clyveycui) [inline src/test/java/seedu/anilist/logic/parser/AddCommandParserTest.java:39]: final variable should be all caps

File: src/test/java/seedu/anilist/logic/parser/DeleteCommandParserTest.java [hunk 21]
```diff
@@ -20,7 +20,7 @@
  */
 public class DeleteCommandParserTest {
 
-    private DeleteCommandParser parser = new DeleteCommandParser();
+    private final DeleteCommandParser parser = new DeleteCommandParser();
```
Comment (clyveycui) [inline src/test/java/seedu/anilist/logic/parser/DeleteCommandParserTest.java:23]: final variable should be all caps

File: src/test/java/seedu/anilist/logic/commands/UpdateEpisodeCommandTest.java [hunk 22]
```diff
@@ -26,7 +27,7 @@
 
 public class UpdateEpisodeCommandTest {
 
-    private Model model = new ModelManager(getTypicalAnimeList(), new UserPrefs());
+    private final Model model = new ModelManager(getTypicalAnimeList(), new UserPrefs());
```
Comment (clyveycui) [inline src/test/java/seedu/anilist/logic/commands/UpdateEpisodeCommandTest.java:30]: final variable should be all caps

File: src/test/java/seedu/anilist/logic/commands/RenameCommandTest.java [hunk 23]
```diff
@@ -25,7 +26,7 @@
 import seedu.anilist.testutil.NameDescriptorBuilder;
 
 public class RenameCommandTest {
-    private Model model = new ModelManager(getTypicalAnimeList(), new UserPrefs());
+    private final Model model = new ModelManager(getTypicalAnimeList(), new UserPrefs());
```
Comment (clyveycui) [inline src/test/java/seedu/anilist/logic/commands/RenameCommandTest.java:29]: final variable should be all caps

File: src/test/java/seedu/anilist/logic/commands/HelpCommandTest.java [hunk 24]
```diff
@@ -9,12 +9,12 @@
 import seedu.anilist.model.ModelManager;
 
 public class HelpCommandTest {
-    private Model model = new ModelManager();
-    private Model expectedModel = new ModelManager();
+    private final Model model = new ModelManager();
+    private final Model expectedModel = new ModelManager();
```
Comment (clyveycui) [inline src/test/java/seedu/anilist/logic/commands/HelpCommandTest.java:13]: final variable should be all caps

File: src/test/java/seedu/anilist/logic/commands/GenreListCommandTest.java [hunk 25]
```diff
@@ -9,8 +9,8 @@
 import seedu.anilist.model.ModelManager;
 
 public class GenreListCommandTest {
-    private Model model = new ModelManager();
-    private Model expectedModel = new ModelManager();
+    private final Model model = new ModelManager();
+    private final Model expectedModel = new ModelManager();
```
Comment (clyveycui) [inline src/test/java/seedu/anilist/logic/commands/GenreListCommandTest.java:13]: final variable should be all caps

File: src/test/java/seedu/anilist/logic/commands/GenreDeleteCommandTest.java [hunk 26]
```diff
@@ -35,7 +36,7 @@
 public class GenreDeleteCommandTest {
     private static final String ANIME_ONE_GENRE = FIRST_ANIME_GENRE;
 
-    private Model model = new ModelManager(getTypicalAnimeList(), new UserPrefs());
+    private final Model model = new ModelManager(getTypicalAnimeList(), new UserPrefs());
```
Comment (clyveycui) [inline src/test/java/seedu/anilist/logic/commands/GenreDeleteCommandTest.java:39]: final variable should be all caps

File: src/test/java/seedu/anilist/logic/commands/GenreAddCommandTest.java [hunk 27]
```diff
@@ -36,7 +37,7 @@
 public class GenreAddCommandTest {
     private static final String ANIME_ONE_GENRE = FIRST_ANIME_GENRE;
 
-    private Model model = new ModelManager(getTypicalAnimeList(), new UserPrefs());
+    private final Model model = new ModelManager(getTypicalAnimeList(), new UserPrefs());
```
Comment (clyveycui) [inline src/test/java/seedu/anilist/logic/commands/GenreAddCommandTest.java:40]: final variable should be all caps

File: src/test/java/seedu/anilist/logic/commands/FindCommandTest.java [hunk 28]
```diff
@@ -31,8 +30,8 @@
  * Contains integration tests (interaction with the Model) for {@code FindCommand}.
  */
 public class FindCommandTest {
-    private Model model = new ModelManager(getTypicalAnimeList(), new UserPrefs());
-    private Model expectedModel = new ModelManager(getTypicalAnimeList(), new UserPrefs());
+    private final Model model = new ModelManager(getTypicalAnimeList(), new UserPrefs());
+    private final Model expectedModel = new ModelManager(getTypicalAnimeList(), new UserPrefs());
```
Comment (clyveycui) [inline src/test/java/seedu/anilist/logic/commands/FindCommandTest.java:34]: final variable should be all caps

File: src/test/java/seedu/anilist/logic/commands/ExitCommandTest.java [hunk 29]
```diff
@@ -9,8 +9,8 @@
 import seedu.anilist.model.ModelManager;
 
 public class ExitCommandTest {
-    private Model model = new ModelManager();
-    private Model expectedModel = new ModelManager();
+    private final Model model = new ModelManager();
+    private final Model expectedModel = new ModelManager();
```
Comment (clyveycui) [inline src/test/java/seedu/anilist/logic/commands/ExitCommandTest.java:13]: final variable should be all caps

File: src/test/java/seedu/anilist/logic/commands/DeleteCommandTest.java [hunk 30]
```diff
@@ -24,7 +25,7 @@
  */
 public class DeleteCommandTest {
 
-    private Model model = new ModelManager(getTypicalAnimeList(), new UserPrefs());
+    private final Model model = new ModelManager(getTypicalAnimeList(), new UserPrefs());
```
Comment (clyveycui) [inline src/test/java/seedu/anilist/logic/commands/DeleteCommandTest.java:28]: final variable should be all caps

**Review by clyveycui — COMMENTED** [review]
Please take a look at the comments

File: src/main/java/seedu/anilist/model/util/SampleDataUtil.java [hunk 31]
```diff
@@ -78,6 +78,7 @@ public static ReadOnlyAnimeList getSampleAnimeList() {
      */
     public static Set<Genre> getGenreSet(String... strings) {
         return Arrays.stream(strings)
+                .map(String::toLowerCase)
```
[the remark this answers is not in the document]
Comment (auhc99) [reply src/main/java/seedu/anilist/model/util/SampleDataUtil.java:81]: Yes, without this alot of test cases will fail

File: src/main/java/seedu/anilist/model/anime/Anime.java [hunk 32]
```diff
@@ -86,8 +86,12 @@ public boolean equals(Object other) {
 
         Anime otherAnime = (Anime) other;
         Name otherAnimeName = otherAnime.getName();
+        Episode otherAnimeEpisode = otherAnime.getEpisode();
+        Status otherAnimeStatus = otherAnime.getStatus();
         Set<Genre> otherAnimeGenres = otherAnime.getGenres();
         return otherAnimeName.equals(this.getName())
+                && otherAnimeEpisode.equals(this.getEpisode())
+                && otherAnimeStatus.equals(this.getStatus())
                 && otherAnimeGenres.equals(this.getGenres());
```
[the remark this answers is not in the document]
Comment (auhc99) [reply src/main/java/seedu/anilist/model/anime/Anime.java:95]: ok

File: src/test/java/seedu/anilist/logic/commands/ActionTest.java [hunk 33]
```diff
@@ -18,16 +22,21 @@ public void isValidActionTest() {
...
-        assertFalse(Action.isValidAction(INVALID_ACTION_EMPTY_STRING));
+        assertFalse(Action.isValidAction(INVALID_ACTION_ALPHA));
+        assertFalse(Action.isValidAction(INVALID_STRING_SPACE));
+        assertFalse(Action.isValidAction(INVALID_STRING_EMPTY));
+        assertFalse(Action.isValidAction(INVALID_ACTION_NUMERIC));
 
         assertTrue(Action.isValidAction(VALID_ACTION_ADD));
+        assertTrue(Action.isValidAction(VALID_ACTION_ADD_SHORT_FORM));
+        assertTrue(Action.isValidAction(VALID_ACTION_DELETE));
+        assertTrue(Action.isValidAction(VALID_ACTION_DELETE_SHORT_FORM));
```
[the remark this answers is not in the document]
Comment (auhc99) [reply src/test/java/seedu/anilist/logic/commands/ActionTest.java:33]: All of them have different cases tho

File: src/test/java/seedu/anilist/logic/commands/ActionTest.java [hunk 34]
```diff
@@ -18,16 +22,21 @@ public void isValidActionTest() {
         // null action string
         assertThrows(NullPointerException.class, () -> Action.isValidAction(null));
 
-        assertFalse(Action.isValidAction(INVALID_ACTION_NO_SUCH_ACTION));
-        assertFalse(Action.isValidAction(INVALID_ACTION_EMPTY_STRING));
+        assertFalse(Action.isValidAction(INVALID_ACTION_ALPHA));
+        assertFalse(Action.isValidAction(INVALID_STRING_SPACE));
+        assertFalse(Action.isValidAction(INVALID_STRING_EMPTY));
+        assertFalse(Action.isValidAction(INVALID_ACTION_NUMERIC));
```
[the remark this answers is not in the document]
Comment (auhc99) [reply src/test/java/seedu/anilist/logic/commands/ActionTest.java:28]: ok, just thought that those invalid actions were all different in terms of being only alphabets, numbers, etc.

File: src/test/java/seedu/anilist/logic/LogicManagerTest.java [hunk 35]
```diff
@@ -40,7 +40,7 @@
     @TempDir
     public Path temporaryFolder;
 
-    private Model model = new ModelManager();
+    private final Model model = new ModelManager();
```
[the remark this answers is not in the document]
Comment (auhc99) [reply src/test/java/seedu/anilist/logic/LogicManagerTest.java:43]: ok

[the remark this answers is not in the document]
Comment (auhc99) [reply src/test/java/seedu/anilist/logic/LogicManagerTest.java:43]: um doing this triggers checkstyle error tho, like we cannot name it all CAPS according to checkstyle

[the remark this answers is not in the document]
Comment (clyveycui) [reply src/test/java/seedu/anilist/logic/LogicManagerTest.java:43]: ok yeah ignore the final stuff
I think it is only required when it is a static final

File: src/test/java/seedu/anilist/logic/commands/AddCommandTest.java [hunk 36]
```diff
@@ -41,7 +42,7 @@ public void execute_animeAcceptedByModel_addSuccessful() throws Exception {
         CommandResult commandResult = new AddCommand(validAnime).execute(modelStub);
 
         assertEquals(String.format(AddCommand.MESSAGE_SUCCESS, validAnime), commandResult.getFeedbackToUser());
-        assertEquals(Arrays.asList(validAnime), modelStub.animesAdded);
+        assertEquals(List.of(validAnime), modelStub.animesAdded);
```
[the remark this answers is not in the document]
Comment (auhc99) [reply src/test/java/seedu/anilist/logic/commands/AddCommandTest.java:45]: nope, one is shorter

File: src/test/java/seedu/anilist/logic/commands/CommandResultTest.java (hunk 1 above)
Comment (auhc99) [reply src/test/java/seedu/anilist/logic/commands/CommandResultTest.java:33]: ok

File: src/test/java/seedu/anilist/logic/commands/CommandTestUtil.java (hunk 2 above)
Comment (auhc99) [reply src/test/java/seedu/anilist/logic/commands/CommandTestUtil.java:50]: ok

File: src/test/java/seedu/anilist/logic/commands/CommandsListTest.java (hunk 3 above)
Comment (auhc99) [reply src/test/java/seedu/anilist/logic/commands/CommandsListTest.java:43]: yeap

File: src/test/java/seedu/anilist/logic/commands/ConfirmClearCommandTest.java (hunk 4 above)
Comment (auhc99) [reply src/test/java/seedu/anilist/logic/commands/ConfirmClearCommandTest.java:25]: YAH SO WE CANT USE ALL CAPS FOR CONSTANTS IN TESTS FOR SOME REASON

File: src/test/java/seedu/anilist/logic/commands/CommandResultTest.java (hunk 1 above)
Comment (auhc99) [reply src/test/java/seedu/anilist/logic/commands/CommandResultTest.java:33]: added constants, but cant name them all caps.

File: src/test/java/seedu/anilist/logic/parser/ParserUtilTest.java (hunk 14 above)
Comment (auhc99) [reply src/test/java/seedu/anilist/logic/parser/ParserUtilTest.java:195]: was debugging, removed

File: src/test/java/seedu/anilist/model/anime/EpisodeTest.java (hunk 9 above)
Comment (auhc99) [reply src/test/java/seedu/anilist/model/anime/EpisodeTest.java:58]: reduced the invalid strings, as for the invalid numbers i think we should still include them

File: src/test/java/seedu/anilist/model/anime/NameTest.java (hunk 8 above)
Comment (auhc99) [reply src/test/java/seedu/anilist/model/anime/NameTest.java:53]: ok

File: src/test/java/seedu/anilist/model/anime/StatusEqualsPredicateTest.java (hunk 7 above)
Comment (auhc99) [reply src/test/java/seedu/anilist/model/anime/StatusEqualsPredicateTest.java:19]: um 'watching' is the default status, so this change is to check if changing this to a non default status would affect anything.

**Review by clyveycui — APPROVED** [review]

File: src/test/java/seedu/anilist/model/ModelManagerTest.java (hunk 10 above)
Comment (auhc99) [reply src/test/java/seedu/anilist/model/ModelManagerTest.java:127]: actually I have changed all back to assertFalse() / assertTrue() as using assertEquals()/ assertNotEquals() will bring code coverage down.

File: src/test/java/seedu/anilist/model/genre/GenreTest.java (hunk 5 above)
Comment (auhc99) [reply src/test/java/seedu/anilist/model/genre/GenreTest.java]: added

File: src/test/java/seedu/anilist/model/anime/StatusTest.java (hunk 6 above)
Comment (auhc99) [reply src/test/java/seedu/anilist/model/anime/StatusTest.java]: removed some
